spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -102,6 +104,8 @@ public StructType outputType() {
         action.executeDeleteWith(executorService(maxConcurrentDeletes, "expire-snapshots"));
       }
 
+      action.option("stream-results", Boolean.toString(streamResult));
```

Comment:
We should probably make "stream-results" a constant in the ExpireSnapshotsAction and use it here rather than passing it as a string

Reply:
It already is :) It's just private, while I wish we could just make it package private on iceberg.spark I think it's fine to just make it public in BaseExpireSnapshotsSparkAction